src/linear_algebra/affine_space/independent.lean

```diff
@@ -343,6 +345,32 @@ begin
     { use [hsvi, affine_span_singleton_union_vadd_eq_top_of_span_eq_top p₁ hsvt] } }
 end
 
+variables (k)
+
+/-- Two different points are affinely independent. -/
+lemma affine_independent_of_ne (p : ι → P) {i₁ i₂ : ι} (he : ∀ i, i = i₁ ∨ i = i₂)
+  (hn : p i₁ ≠ p i₂) : affine_independent k p :=
```

Comment:
I'll take your word for it if this is useful elsewhere, but the hypothesis `he : ∀ i, i = i₁ ∨ i = i₂` seems unlikely to occur in the wild. Would it lose anything if you just combine this proof into the next statement?

Reply:
I've combined these two lemmas, since the `fin 2` version is the one I have a use for.